README.md

```diff
@@ -42,28 +43,34 @@
   <ul>
     <li><a href="https://expressjs.com/">Express.js</a></li>
   </ul>
-</details>
+</details> -->
 
-<details>
+<!-- <details>
 <summary>Database</summary>
   <ul>
     <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
-</details>
+</details> --> 
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-- **[]**
-- **[]**
-- **[]**
-- **[]**
+- **List all books**
+- **List all music albums**
+- **List all games**
+- **List all genres**
+- **List all labels**
+- **List all authors**
+- **Add a book**
+- **Add a music album**
+- **Add a game**
+- **A means to Quit the application**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-- [Live Demo Link]()
+- [Live Demo Link](https://drive.google.com/file/d/1sZzqjjENOHRRgpYXsu_HWGJWV4PLKegc/view?usp=sharing)
```

Comment:
- [ ] `required`
- [ ] Your presentation was okay, it's unfortunate your teammate wouldn't join you guys 😭 
- [ ] however, your video presentation doesn't meet the requirements 😞 
- [ ] The requirements state that the video maximum minutes has to be `5 minutes  but you did it `12 minutes `
- [ ] Please meet this requirement 🥂 
- [ ] see screenshot below
![image](https://user-images.githubusercontent.com/66230306/213865226-9f102edb-1743-4750-800c-683df71a6f3b.png)